internal/github/checkStalePRs.go

```diff
@@ -0,0 +1,71 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v60/github"
+
+	"github.com/chia-network/github-bot/internal/config"
+)
+
+// CheckStalePRs will return a list of PR URLs that have not been updated in the last 7 days by internal team members.
+func CheckStalePRs(githubClient *github.Client, internalTeam string, cfg config.LabelConfig) ([]string, error) {
+	var stalePRUrls []string
+	cutoffDate := time.Now().AddDate(0, 0, -7) // 7 days ago
+	teamMembers, err := GetTeamMemberList(githubClient, internalTeam)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, fullRepo := range cfg.LabelCheckRepos {
```

Comment:
`cfg.LabelCheckRepos` should this be its own list in the config file since it's a different check than the label one?